desktop/utils/domain/auctions/urls.js

```diff
@@ -0,0 +1,18 @@
+const sd = require('sharify').data
+const config = require('../../../../config')
+/** Get the live auction url with /login appended when a user is present
+ * @param {String} id - auction id/slug
+ * @param {Object} [options]
+ * @param {Boolean} [options.isLoggedIn] - whether there is a user present
+ * @return {String} live auction url
+ */
+export const liveAuctionUrl = (id, options = {}) => {
+  const liveAuctionRoot = sd.PREDICTION_URL || config.PREDICTION_URL
```

Comment:
It appears that you might be using this util client-side which is why tests are [failing on this line](https://github.com/artsy/force/blob/master/desktop/config.coffee#L112), but more importantly you don't want to require config directly—sharify should be enough because it works server & client. So this line should just be `sd.PREDICTION_URL`. Curious why the need for ` || config.`?